data.sql

```diff
@@ -7,4 +7,11 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Agumon', '2020-02-03', 10.23, true, 0),
 ( 'Gabumon', '2018-11-15', 8.00, TRUE, 2),
 ('Pikachu', '2021-01-07', 15.04, FALSE, 1),
-('Devimon', '2017-05-12', 11.00, TRUE, 5);
\ No newline at end of file
+('Devimon', '2017-05-12', 11.00, TRUE, 5);
+('Charmander', '2020-02-08', -11.00, FALSE, 0),
+('Plantmon', '2021-11-15', -5.70, TRUE, 2),
+('Squirtle', '1993-04-02', -12.13, FALSE, 3),
+('Angemon', '2005-06-12', -45.00, TRUE, 0),
+('Boarmon', '2005-06-07', 20.40, TRUE, 7),
+('Blossom', '1998-09-13', 17.00, TRUE,3),
```

Comment:
Great job @theodoku 🤝

- [ ] As stated in the project requirements, kindly ensure the `date_of_birth` for `Blossom` is not `1998-09-13` but rather `1998/10/13`.

Reply:
date_of_birth for Blossom updated